jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+
+**Note:** The user must explicitly name repeat jobs when a repeat job should be upstream of another job in a workflow (ie: if the job is used under the `requires` key of a job invocation in a workflow you will need to name it).
+
+```yaml
+version: 2.1
+workflows:
+  build:
+    jobs:
+      - loadsay
+      # This doesn't need an explicit name as it has no downstream dependencies
+      - sayhello:
+          saywhat: Everyone
+          requires:
+            - loadsay
+      # This needs an explicit name for saygoodbye to require it as a job dependency
+      - sayhello:
+          name: SayHelloChad
+          saywhat: Chad
+      # Uses explicitly defined "sayhello"
+      - saygoodbye:
+          requires:
+            - SayHelloChad
+```
+
+### Pre and Post Steps
+{:.no_toc}
+
+**Note:** The keys `pre-steps` and `post-steps` in jobs are available in configuration version 2.1 and later_
+
+Every job accepts two special arguments: `pre-steps` and `post-steps`.
+You can optionally invoke a job with one or both of these arguments. Steps under `pre-steps`
+are executed before any of the other steps in the job. The steps under
+`post-steps` are executed after all of the other steps.
+
+For this reason, the parameter names `pre-steps` and `post-steps` are reserved
+and may not be redefined by a job author. For example, the following job
+definition is invalid:
+
+```yaml
+jobs:
+  foo:
+    parameters:
+      pre-steps:    # invalid: pre-steps is a reserved parameter name
+        type: steps
+        default: []
+```
+
+All jobs accept two special arguments of type `steps`: `pre-steps` and
+`post-steps`.
+
+If an orb user invokes a job with one or both of these arguments,
+the job will run the steps in `pre-steps` first, before any other steps run, and
+then it will run the steps in `post-steps` last, after any other steps run.
+
+Pre- and post- steps allow users to be execute steps in a given job's environment
+without modifying the orb. This is useful, for example, when a user imports a job
+and wants to upload assets after it completes, or to run some custom setup steps
+before job execution. Pre- and post- steps allow the user to make these additions
+without modifying the imported job.
+
+A `steps` parameter can be used for a similar purpose, passing steps into a job,
+but it requires that the job be modified with an execution site for the parameter.
+
+#### Example of Using Pre- and Post-Steps
+{:.no_toc}
+
+An orb `foo` might define a job:
+
+```yaml
+# yaml from orb `foo`
+jobs:
+  bar:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          command: echo "building"
+      - run:
+          command: echo "testing"
+```
+
+Then an orb user could use the job as follows:
+```yaml
+# config.yml
+version: 2.1
+orbs:
```

Comment:
I thought we were planning to not include orbs yet? If so, let's not have examples with them as it might confuse the concepts here.